bigquery/snippets/table/bigquery_create_table_range_partitioned.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
```

Comment:
This makes me curious, is there a header check on this repo?

Reply:
I believe it's in the kokoro config, but it can't reason about the year for a given delta since it sees a snapshot.  Corrected.